InfinityCosmos/ForMathlib/AlgebraicTopology/SimplicialSet/Monoidal.lean

```diff
@@ -0,0 +1,6 @@
+import Mathlib.AlgebraicTopology.SimplicialSet.Monoidal
+import InfinityCosmos.ForMathlib.AlgebraicTopology.SimplexCategory
+
+def SSet.isTerminal : IsTerminal (Δ[0] : SSet.{u}) where
```

Comment:
Perhaps this term should be called `SSet.isTerminalDeltaZero`. Or you could put the def in the `SSet` namespace and remove the "SSet."